content/home/examples/a-simple-component.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: A Simple Component
+title: Basit Bir Bileşen
 order: 0
 domid: hello-example
 ---
 
-React components implement a `render()` method that takes input data and returns what to display. This example uses an XML-like syntax called JSX. Input data that is passed into the component can be accessed by `render()` via `this.props`.
+React bileşenleri, girdi verileri alan ve ne gösterileceğini dönen bir `render()` methodu sağlar. Bu örnekte, XML benzeri bir sentaks olan JSX kullanılıyor. Bileşene iletilen girdi verilerine, `render()` methodunda `this.props` aracılığıyla ulaşılabilir.
```

Comment:
What about just `syntax` or `söz dizimi` ?

Reply:
Instead of `syntax`, I'd be more okay with `söz dizimi`. However when I check TDK, I can see `sentaks` is already included in the dictionary.